app/assets/stylesheets/modules/show_playdates.scss

```diff
@@ -0,0 +1,25 @@
+html body {
+  height: 100%;
+  margin: 0;
+}
+
+#map {
+  border-style: solid;
+  border-width: 7px;
+  box-shadow: 2px 2px 10px #B1B1B1;
```

Comment:
Color literals like `#B1B1B1` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#B1B1B1` should be written as `#b1b1b1`